<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selldetails</class>
 <widget class="QDialog" name="selldetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Billing</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #bfe0f4;</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Bill No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_billno">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(85, 85, 255);</string>
   </property>
   <property name="title">
    <string>Bill Info</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>Drug ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>[ ]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_drugid">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_quantity">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>Discount</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_discount">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
    </property>
    <property name="text">
     <string>Item Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>Quanty</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>Available Stock:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_totalamnt">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>590</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Calculate total Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>600</y>
     <width>55</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Total</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>591</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_reset">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_print">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>600</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Customer Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_customername">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:#2c2255;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveinfo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Save Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Display/Refresh</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
